### PR TITLE
add custom gutter to grid

### PR DESCRIPTION
**Pull request by JoanClaret**
- Modify grid system in order to be able to use custom gutters.
- Add `$gutter-s` to be used in grids with a small gutter.

### CODE REVIEW

File: src/layout/_grid.scss [hunk 1]
```diff
@@ -10,9 +10,14 @@
...
 // @include sui-cell-of(2);
+//
+// Custom gutter:
+// @include sui-grid($gutter: $custom-gutter)
+// @include sui-cell-of($col-num, $gutter: $custom-gutter)
+
 
 // Grid
-@mixin sui-grid ($add-gutter: true, $inline: false) {
+@mixin sui-grid ($add-gutter: true, $inline: false, $gutter: $gutter) {
```
[the remark this answers is not in the document]
Comment (JoanClaret) [reply src/layout/_grid.scss]: ok, no problem. anyways we need to add the custom gutter like so:

// draft code
@mixin sui-grid ($add-gutter: true, $inline: false, $gutter: $gutter) {
    @if type-of($add-gutter) != bool {
       $gutter = $add-gutter
    }
    @if $add-gutter != false {
       padding: 0 $gutter $gutter;
    }
    [...]
}

[the remark this answers is not in the document]
Comment (JoanClaret) [reply src/layout/_grid.scss]: To be honest, looks  a bit weird to me. 
Any others opinions? @SUI-Components/developers

[the remark this answers is not in the document]
Comment (betobe) [reply src/layout/_grid.scss]: +1 for Joan's code

[the remark this answers is not in the document]
Comment (kikoruiz) [reply src/layout/_grid.scss]: Yep, I prefer the code from the PR too. Don't like too much mixing types for arguments.

[the remark this answers is not in the document]
Comment (JoanClaret) [reply src/layout/_grid.scss]: ok then, i'll merge this PR as is. we'll see in the future if we need to refactor it.
Thank you everyone!

**Pull request merged**